### PR TITLE
Features/mip vcfparser

### PR DESCRIPTION
**Pull request by northwestwitch**
Moved Mip (vcfparser) to MIP::Program::Variantcalling::Mip_vcfparser (mip_vcfparser). Tested on rasta.

### CODE REVIEW

File: lib/MIP/Program/Variantcalling/Mip_vcfparser.pm [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+    my $tmpl = {
+        FILEHANDLE => {
+            store => \$FILEHANDLE,
+        },
+        infile_path => {
+            required    => 1,
+            defined     => 1,
+            strict_type => 1,
+            store       => \$infile_path
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Mip_vcfparser.pm]: \$infile_path -> \$infile_path,
throughout

File: lib/MIP/Program/Variantcalling/Mip_vcfparser.pm [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+        },
+
+        #outfile_path => { strict_type => 1, store => \$outfile_path },
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Mip_vcfparser.pm]: Remove

File: lib/MIP/Program/Variantcalling/Mip_vcfparser.pm [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+        push @commands, q{--parse_vep};
+    }
+
+    if ($per_gene) {
+
+        push @commands, q{--per_gene};
+    }
+
+    if ( defined $padding ) {
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Mip_vcfparser.pm]: I think you can remove defined

File: lib/MIP/Program/Variantcalling/Mip_vcfparser.pm [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+        push @commands, q{--padding} . $SPACE . $padding;
+    }
+
+    if ($range_feature_file_path) {
+
+        push @commands,
+          q{--range_feature_file} . $SPACE . $range_feature_file_path;
+    }
+
+    #Limit output to regions
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Mip_vcfparser.pm]: '#' Limit
throughout

**Comment by henrikstranneheim** [comment]
👍 
Fix the merge conflict and then I will merge this

**Comment by henrikstranneheim** [comment]
Great!

**Comment by northwestwitch** [comment]
Made a pull from your the official develop, I realize now that it was better not to do it because it looks more messy. But basically this pull request has also the other updates. I've removed the Program::Variantcalling altogether because it was now empty.

**Comment by henrikstranneheim** [comment]
Should be ok.

**Comment by henrikstranneheim** [comment]
Now boarding for Paris! See you on Tuesday!

**Comment by northwestwitch** [comment]
Nice, enjoy the trip! :)

**Pull request merged**